internal/app/app_dcs.go

```diff
@@ -136,3 +136,102 @@ func (app *App) GetLastShutdownNodeTime() (time.Time, error) {
 	}
 	return t, err
 }
+
+// Finish current switchover and write the result
+func (app *App) finishSwitchover(switchover *Switchover, switchErr error) error {
```

Comment:
These methods should be public, because one day we will extract AppDcs interface